vumi_message_store/tests/helpers.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime, timedelta
+
+from twisted.internet.defer import inlineCallbacks, returnValue
+from zope.interface import implements
+
+from vumi.tests.helpers import IHelper
+from vumi.message import format_vumi_date
+
+
+class MessageSequenceHelper(object):
+    implements(IHelper)
```

Comment:
Rather use the `@implementer` class decorator instead of `implements()`. It's less magic and more obvious about what it's doing.